Formula/llvm.rb

```diff
@@ -137,13 +139,29 @@ def install
       -DCLANG_PYTHON_BINDINGS_VERSIONS=#{py_ver}
     ]
 
-    sdk = MacOS.sdk_path_if_needed
-    args << "-DDEFAULT_SYSROOT=#{sdk}" if sdk
+    on_macos do
+      args << "-DLLVM_LINK_LLVM_DYLIB=ON"
```

Comment:
This isn't platform specific. It's what makes LLVM link to itself via shared libraries and makes a massive difference on bottle size.

Reply:
Yep, you definitely want this flag: https://llvm.org/docs/BuildingADistribution.html#options-for-reducing-size